### ISSUE TITLE
Potential lazy regex in default value if badly edited in the future + useless part

### ISSUE THREAD

**Issue by Starmania** [pylint-dev/pylint#8913]
### Bug description

In the default config file, for the `ignore-long-lines` regex, where the default value is `^\s*(# )?<?https?://\S+>?$`.

If someone decide to modifie this value, it is really easy to create a lazy regex expression by removing the last `?` and adding anything between the `>` and the `$`.

Also, the last `>` is useless, because the `\S+` will always match this character already. 

Last but not least, this regex is only useful on comments. I think this should be more open to any url in general, as even basic one could be really long quickly.

### Command used

```shell
n\a
```

### Pylint output

```shell
n\a
```

### Expected behavior

Recognize that a simple url that is simple in is construction should be accepted.

And get an extra boost of performance by skipping one token (big perf....). 

### Pylint version

```shell
pylint 2.17.4
astroid 2.15.4
Python 3.11.4 (tags/v3.11.4:d2340ef, Jun  7 2023, 05:45:37) [MSC v.1934 64 bit (AMD64)]
```

**Comment by Starmania** [comment]
Sorry, because after checking what I wrote, I just realize that my bug report become a suggestion...

**Comment by DanielNoord** [comment]
I don't think we would want to change this value. It's over writable if users want to and changing default values can break setups in annoying ways. Only the duplicate token can be removed.

Perhaps you would like to contribute a documentation improvement to include examples how to improve the regex for common use cases?
